<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomPropertyEditorDialog</class>
 <widget class="QDialog" name="CustomPropertyEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Custom Property Editor</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QtnPropertyWidgetEx" name="propertyWidget" native="true">
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionPropertyAdd">
   <property name="text">
    <string>Add Child Property...</string>
   </property>
   <property name="toolTip">
    <string>Add Child Property</string>
   </property>
  </action>
  <action name="actionPropertyRemove">
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="toolTip">
    <string>Remove Property</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionPropertyDuplicate">
   <property name="text">
    <string>Duplicate...</string>
   </property>
   <property name="toolTip">
    <string>Duplicate Property</string>
   </property>
  </action>
  <action name="actionPropertyOptions">
   <property name="text">
    <string>Options...</string>
   </property>
   <property name="toolTip">
    <string>Property Options</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtnPropertyWidgetEx</class>
   <extends>QWidget</extends>
   <header>QtnProperty/Extension/PropertyWidgetEx.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
